web/yo/app/views/drugs.html

```diff
@@ -0,0 +1,39 @@
+<div class="drugs">
+        <h3 class="headerStyle">Curated Drugs</h3>
+        <table class="table" datatable="ng" dt-options="dtOptions" dt-columns="dtColumns">
+            <thead>
+            <tr class="dashedHeader">
+                <th>Name</th>
+                <th>NCI Thesaurus Code</th>
+                <th>Description</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr data-ng-repeat="drug in drugList">
+                <td ng-include="getTemplate(drug)">
+                </td>
+                <td>{{ drug.NCI }}</td>
+                <td>{{ drug.description }}</td>
+            </tr>
+            </tbody>
+        </table>
+
+        <div class="col-sm-9">
```

Comment:
Is it really necessary to use sm-9 for the dropdown?

Reply:
Has changed it to fixed width.